src/tulip/index.ts

```diff
@@ -155,5 +249,13 @@ export interface DepositorInfo extends IDepositorInfo {
   totalWithdrawnUnderlying: BN;
   lastPendingReward: BN;
   rewardPerSharePaid: BN;
-  extra_data_account: PublicKey;
+  extraDataAccount: PublicKey;
+}
+
+export enum VaultType {
+  Raydium,
+  Orca,
+  OrcaDD,
+  LendingOptimizer,
```

Comment:
How about `Saber` and `Atrix`

Reply:
Tulip team said they don't support Saber and Atrix any more.

Reply:
Noted